nemo_skills/inference/server/model.py

```diff
@@ -656,8 +715,9 @@ def _generate_single(
                 seed=random_seed,
                 stop=stop_phrases,
                 messages=prompt,
+                logprobs=logprobs is not None,
+                top_logprobs=logprobs,
```

Comment:
Let's perhaps call this parameter top_logprobs everywhere? Simply logprobs with integer type is confusing